applications/Bifrost-PLO-Liquidity.md

```diff
@@ -0,0 +1,153 @@
+# Open Grant Proposal
+
+* **Project Name:** Bifrost PLO Liquidity
+* **Team Name:** Bifrost.Finance
+* **Payment Address:** 0x3754Fd5eC997c07Ca528bb0bdCB0b4F5b93605dF
+
+## Project Description
+Bifrost is a parachain that provides liquidity for staking and collateral assets. It is positioned as the defi protocol in the Polkadot ecology. Based on the development of substrate, the parachain slot will be auctioned. Staking is the first kind of asset for Bifrost to unlock liquidity. Users can enjoy staking rewards and liquidity both through Bifrost protocol by mint vToken in PoS.
+
+Bifrost plans to provide KSM/DOT liquidity for the Kusama/Polkadot parachain slot auction, because of the limited number of Parachain Slots, the initial price of parachain slots are expected to be expensive, and projects who need to lease parachain will conduct the Parachain Lease Offering (PLO) to gain their fund advantage in the slot auction. Therefore, a large number of KSMs/DOTs will be locked into the parachain slot auction, and users who contribute their KSM/DOT into parachian crownloan are willing to face a period of time (6-24 months if the parachain wins) that their KSM/DOT cannot be redeemed, which means users will not be able to trade the assets and take the oppotunity cost that the lock-up period without staking rewards. 
+
+However, currently users can participate in the parachain slot auction through Bifrost by this proposal, and obtain the corresponding vsDOT (Voucher Slot DOT) or vsKSM to unlock liquidity. vsKSM/vsDOT is fungible asset that represents all parachains' contributors KSM/DOT, with winner parachain's crowdloan rewards and the derivative is tradable. When users deposit KSM/DOT for a parachain's crownloan throught Bifrost protocol, they can get vsDOT/vsKSM and a certification that vertify the holder of certification can obtain the parachain's rewards, but vsDOT/vsKSM and certifications are not tradable yet until the parachian project wins the auction. If the project lose, those vsDOT/vsKSM and certifications will be eliminated automatically. When users want to cash out their lock-up KSM/DOT after the parachian wins the action, they can sell vsDOT or vsKSM on market with market price. Or they can hold it until the parachain slot expired, and redeem their original assets by certifications. 
+
+The trade module has been developed in advance, and vsDOT/vsKSM will be the second type of collateral assets' derivative supported by Bifrost Finance.
```

Comment:
What does the trade module do?

Reply:
Trade module means DEX module, Balancer, Uniswap, or Zenlink...